onedal/ensemble/forest.py

```diff
@@ -302,7 +302,7 @@ def _validate_targets(self, y, dtype):
         return _column_or_1d(y, warn=True).astype(dtype, copy=False)
 
     def _get_sample_weight(self, X, y, sample_weight):
-        n_samples, _ = X.shape
+        n_samples = X.shape[0]
```

Comment:
Current variant directly checks for the number of dimensions

Reply:
Previous checked it in another way while unpacking shape